msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/HttpHelper.java

```diff
@@ -102,6 +102,8 @@ private static IHttpResponse executeHttpRequestWithRetries(HttpRequest httpReque
             }
             Thread.sleep(RETRY_DELAY_MS);
         }
+
+        log.info(httpResponse.body());
```

Comment:
Please do not log HTTP responses, as they contain tokens. Not even as PII. This is a blocking comment.

Reply:
Accidental check in